XenAdmin/Commands/NewTemplateFromSnapshotCommand.cs

```diff
@@ -65,15 +68,16 @@ public NewTemplateFromSnapshotCommand(IMainWindow mainWindow, VM snapshot)
 
         protected override void RunCore(SelectedItemCollection selection)
         {
-            VM snapshot = (VM)selection[0].XenObject;
+            var snapshot = (VM)selection[0].XenObject;
 
             // Generate list of all taken VM/snapshot/template names
-            List<string> takenNames = new List<VM>(snapshot.Connection.Cache.VMs).ConvertAll(v => v.Name());
+            var takenNames = new List<VM>(snapshot.Connection.Cache.VMs).ConvertAll(v => v.Name());
 
             // Generate a unique suggested name for the new template
-            string defaultName = Helpers.MakeUniqueName(String.Format(Messages.TEMPLATE_FROM_SNAPSHOT_DEFAULT_NAME, snapshot.Name()), takenNames);
+            var defaultName = Helpers.MakeUniqueName(string.Format(Messages.TEMPLATE_FROM_SNAPSHOT_DEFAULT_NAME, snapshot.Name()), takenNames);
 
-            using (var dialog = new InputPromptDialog {
+            using (var dialog = new InputPromptDialog
+            {
                 Text = Messages.SAVE_AS_TEMPLATE,
                 PromptText = Messages.NEW_TEMPLATE_PROMPT,

```

Comment:
Not caused by this PR, but since we're here can we add articles to this message: `Enter a &amp;name for the new template:`